pkg/manifest/manifest_loader.go

```diff
@@ -225,7 +225,6 @@ func parseOAuth(a oAuth) (OAuth, error) {
 	if a.TokenEndpoint == nil {
 		urlDef = URLDefinition{
 			Value: endpoints.Dynatrace.TokenURL,
-			Type:  Absent,
```

Comment:
nit: I would still define it here, to be explicit

Reply:
done